lib/ui/collections_gallery_widget.dart

```diff
@@ -691,7 +690,9 @@ class SectionTitle extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      margin: EdgeInsets.fromLTRB(24, 12, 0, 0),
+      // margin: EdgeInsets.fromLTRB(24, 12, 0, 0),
+      margin: EdgeInsets.fromLTRB(
+          (title == "Incoming" || title == "Outgoing") ? 16 : 24, 12, 0, 0),
```

Comment:
This check seems brittle. Perhaps let the wrapping container define the margins?

Reply:
Have removed the check completely and and left padding is now 16 everytime.